src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/Startup.cs

```diff
@@ -87,17 +99,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 #endif
             app.UseStaticFiles();
             app.UseSpaStaticFiles();
-#if (IndividualLocalAuth)
 
+            app.UseRouting();
+
+#if (IndividualLocalAuth)
             app.UseAuthentication();
             app.UseIdentityServer();
 #endif
+            app.UseAuthorization();
```

Comment:
I think authorization should only get included when you aren't doing any authentication.